src/Maestro/Client/src/MaestroApiCredential.cs

```diff
@@ -51,26 +54,74 @@ public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext _, Canc
         /// <summary>
         /// Use this for user-based flows (darc invocation from dev machines).
         /// </summary>
-        internal static MaestroApiCredential CreateUserCredential(string barApiBaseUri)
+        internal static MaestroApiCredential CreateUserCredential(string barApiBaseUri, string cachePath = "")
```

Comment:
Yeah, we should use `null` instead

Reply:
Discussed offline, the param is no longer optional